pkg/broker/user_info.go

```diff
@@ -0,0 +1,62 @@
+package broker
+
+import (
+	"encoding/json"
+	"fmt"
+
+	osb "github.com/pmorie/go-open-service-broker-client/v2"
+)
+
+// ParseKubernetesIdentity - creates a kubernetes identity from the
+// orginating identity
+func ParseKubernetesIdentity(o osb.OriginatingIdentity) (*K8SIdentity, error) {
+	if o.Platform != "kubernetes" {
+		return nil, fmt.Errorf("invalid plateform value, expecting kubernetes")
+	}
+
+	u := UserInfo{}
+	err := json.Unmarshal([]byte(o.Value), &u)
+	if err != nil {
+		return nil, fmt.Errorf("unable to unmarshal json for value")
+	}
+	return &K8SIdentity{UserInfo: u, Platform: o.Platform}, nil
+}
+
+// ParseCloudFoundryIdentity - creates a cloud foundry identity from the
+// orginating identity
+func ParseCloudFoundryIdentity(o osb.OriginatingIdentity) (*CloudFoundryIdentity, error) {
+	if o.Platform != "cloudfoundry" {
+		return nil, fmt.Errorf("invalid platform value, expecting cloudfoundry")
```

Comment:
Same as above: Would it be helpful to maybe include in the error msg which platform it was?